src/integration_tests/mission_takeoff_land.cpp

```diff
@@ -73,18 +73,18 @@ void do_mission_with_takeoff_and_land(float mission_altitude_m)
         Telemetry::LandedState::TakingOff,
         Telemetry::LandedState::InAir,
         Telemetry::LandedState::Landing,
-        Telemetry::LandedState::OnGroud};
+        Telemetry::LandedState::OnGround};
 
     Telemetry::LandedState landed_states[8];
     int index = 0;
-    telemetry->subscribe_landed_state([](Telemetry::LandedState landed_state) {
-        LogInfo() << landed_state;
-        landed_states[index] = landed_state;
-        index++
-    })
+    telemetry->subscribe_landed_state(
+        [&index, &landed_states](Telemetry::LandedState landed_state) {
```

Comment:
I am worried about &index. As I understand C++, &index is address of int value. Is it problem, then to do index++?

Reply:
It's correct :+1:

Reply:
Note that instead of using arrays, you could have used vectors. But I believe this works.

Reply:
Modified PR to vectors. Thanks for your advices ;)